armi/reactor/converters/axialExpansion/tests/test_expansionData.py

```diff
@@ -0,0 +1,321 @@
+import unittest
```

Comment:
We are missing a license header here.

Reply:
https://github.com/terrapower/armi/pull/1435/commits/acd932bc13ebb161a0a3f590e50f5e39ebd862a6